client/keys.go

```diff
@@ -658,7 +654,7 @@ func unmarshalHTTPResponse(code int, header http.Header, body []byte) (res *Resp
 
 func unmarshalSuccessfulKeysResponse(header http.Header, body []byte) (*Response, error) {
 	var res Response
-	err := codec.NewDecoderBytes(body, new(codec.JsonHandle)).Decode(&res)
+	err := CaseSensitiveJsonIterator().Unmarshal(body, &res)
```

Comment:
can we reuse a private iterator instance?

Reply:
ack doing that now.